### PR TITLE
add documentation for basic NAT setup

### PR DESCRIPTION
**Pull request by sduthil**

### CODE REVIEW

File: content/uc-doc/administration/nat.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+A call is established between the phone and the Wazo Platform, but the phone can't hear any voice coming in.
+
+Explanation: Since the phones are in a private network, they only know their private address and port, not their public address and port. SIP phones will send their private address and port in the SIP messages, and the Wazo Platform will send the voice packets to the private IP address and port of the phone. However, the private address and port of the phone is not reachable from the Wazo Platform: they aire only valid in the private network of the phone.
+
+
+### The solution
+
+You must add the following option in the affected SIP lines:
+
+* `nat` = `yes`
```
Comment (sboily) [inline content/uc-doc/administration/nat.md:25]: Can we waiting for pjsip native implementation please. nat = yes is only chan_sip.conf and we'll release in a few weeks the full remove of chan_sip. Atnoher options is using the chan_pjsip nat option.

**Comment by sboily** [comment]
The asterisk wiki documentation is very well explained https://wiki.asterisk.org/wiki/display/AST/Configuring+res_pjsip+to+work+through+NAT

**Review by sboily — CHANGES_REQUESTED** [review]
This documentation is now obsolete, nat = yes doesn't exist anymore. @sduthil i can help you to update it if you need. It would be nice to also add paragraph about STUN, TURN etc ... We need to adapt for chan_pjsip because we don't support anymore chan_sip on wazo.

**Comment by fblackburn1** [comment]
@sduthil can we update this PR or simply close it?

File: content/uc-doc/administration/nat.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+  * `rtp_symmetric` = `yes`
+
+  This setting instructs Wazo Platform (via Asterisk) to send the voice packets
+  back to the address and port it came from, i.e. the public address and port of
+  the phone.
+
+* Also make sure that the `global` SIP template has no option `force_rport`. The
+  default value for this option is `yes`, which is the desired value for NAT
+  setups.
```
Comment (sboily) [inline content/uc-doc/administration/nat.md:51]: I think you also need rewrite_contact to have a working solution. Have you tried?

**Review by sboily — CHANGES_REQUESTED** [review]

File: content/uc-doc/administration/nat.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+---
+title: NAT traversal
+---
+
+[//]: # (Asterisk doc: https://wiki.asterisk.org/wiki/display/AST/Configuring+res_pjsip+to+work+through+NAT)
```
Comment (fblackburn1) [inline content/uc-doc/administration/nat.md:5]: What is suppose to do?
The preview display nothing
https://sf.production.wazo.cloud/logs/14/214/9868a328718c938bbf55fedecbdfbe1e94840a1e/check/yarn-build-copy/275ef72/public/uc-doc/administration/nat

**Review by fblackburn1 — CHANGES_REQUESTED** [review]
I guess my approval is only for the styling/form?
You can run `prettier`  to follow the styleguide
```
yarn prettier --print-width 100 --prose-wrap always --write  content/uc-doc/administration/nat.md
```

File: content/uc-doc/administration/nat.md (hunk 3 above)
Comment (sduthil) [reply content/uc-doc/administration/nat.md:5]: It's the least intrusive way to write a comment in Markdown :/
So it's normal nothing is displayed.

File: content/uc-doc/administration/nat.md (hunk 2 above)
Comment (sduthil) [reply content/uc-doc/administration/nat.md]: You're right, I added the option.

**Review by pc-m — APPROVED** [review]

**Pull request merged**